programs/compute-budget/src/lib.rs

```diff
@@ -1,9 +1,6 @@
-#[cfg(feature = "svm-internal")]
-use qualifier_attr::qualifiers;
 use solana_program_runtime::declare_process_instruction;
 
-#[cfg_attr(feature = "svm-internal", qualifiers(pub))]
-const DEFAULT_COMPUTE_UNITS: u64 = 150;
+pub const DEFAULT_COMPUTE_UNITS: u64 = 150;
```

Comment:
this doesn't need to be 'pub', does it?

Reply:
It's being used by `builtins-default-costs.rs`.
```
        compute_budget::id(),
        BuiltinCost::NotMigrating(NotMigratingBuiltinCost {
            native_cost: solana_compute_budget_program::DEFAULT_COMPUTE_UNITS,
        }),
```